src/ray/object_manager/object_manager.cc

```diff
@@ -172,13 +172,16 @@ ray::Status ObjectManager::PullEstablishConnection(const ObjectID &object_id,
               ConnectionPool::ConnectionType::MESSAGE, connection_info);
           connection_pool_.RegisterSender(ConnectionPool::ConnectionType::MESSAGE,
                                           client_id, async_conn);
-          RAY_CHECK_OK(PullSendRequest(object_id, async_conn));
+          Status pull_send_status = PullSendRequest(object_id, async_conn);
+          if (!pull_send_status.ok()) {
+            RAY_LOG(ERROR) << pull_send_status.message();
```

Comment:
Why log the error here, but not after the call to `PullSendRequest` below?

Reply:
Will make this a hard fail.